testing/live/Tf2CriticalHits/manifest.toml

```diff
@@ -1,5 +1,5 @@
 [plugin]
-repository = "https://github.com/Berna-L/ffxiv-tf2-crit-plugin"
+repository = "https://github.com/Berna-L/ffxiv-tf2-crit-plugin.git"
 commit = "f6512969d133ca160b0053bbdca08c4858eae311"
 owners = ["Berna-L"]
 project_path = "/TF2CriticalHitsPlugin"
```

Comment:
the slash is implied, this might complain

Reply:
Removed it, thanks!